GOVERNANCE.md

```diff
@@ -0,0 +1,122 @@
+
+Votes may take the form of a single proposal, with the option to vote yes or no, or the form of multiple alternatives.
+
+A vote on a single proposal is considered successful if at least two thirds of those eligible to vote vote in favor.
+
+If there are multiple alternatives, members may vote for one or more alternatives, or vote "no" to object to all alternatives. A vote on multiple alternatives is considered decided in favor of one alternative if it has received the most votes in favor, and a vote from at least two thirds of those eligible to vote. Should no alternative reach this quorum, another vote on a reduced number of options may be called separately.
+
+## FAQ
+
+This section is informational. In case of disagreement, the rules above overrule any FAQ.
+
+### How do I propose a decision?
+Propose it in the [Fn Project forums](https://forums.fnproject.io/). If there is no objection within a reasonable amount of time, consider the decision made. If there are objections and no consensus can be found, a vote may be called by a team member.
+
+### How  do I become a team member
+To become an official team member, you should make ongoing contributions to one or more project(s) for at least three months. At that point, a team member (typically a maintainer of the project) may propose you for membership. The discussion about this will be held in private, and you will be informed privately when a decision has been made. A possible, but not required, graduation path is to become a maintainer first.
+
+Should the decision be in favor, your new membership will also be announced in the [Fn Project forums](https://forums.fnproject.io/).
+
+### How do I add a project?
+As a team member, propose the new project in the [Fn Project forums](https://forums.fnproject.io/). If nobody objects, create the project in the GitHub organization. Add at least a `README.md` explaining the goal of the project, and a `MAINTAINERS.md` with the maintainers of the project (at this point, this probably means you).
+
+### How do I archive or remove a project?
+
+Start a new topic in the [Fn Project forums](https://forums.fnproject.io/) proposing the retirement of a project. If nobody objects, move it to the fnproject-junkyard GitHub organization.
+
+### How do I remove an inactive maintainer?
+A maintainer may resign by notifying the [Fn Team group on the Fn Project forums](https://forums.fnproject.io/groups/FnTeam). A maintainer with no project activity for a year will be treated as if they had resigned. If there is an urgent need to replace a maintainer, discuss this in the [private Fn Team category in the Fn Project forums](https://forums.fnproject.io/c/fn-team) or by messaging the [Fn Team group](https://forums.fnproject.io/groups/FnTeam).
+
+### How do I remove a team member?
```

Comment:
this seems somewhat in conflict with the prior answer of proposing a supermajority vote https://github.com/fnproject/fn/pull/1094/files#diff-9e8744ec73212c1966bb19fa22fc3e0aR29 -- we should pick one and just put it there instead of having to diff 2 sections I think